ehb_service/apps/core/models/identities.py

```diff
@@ -373,10 +373,11 @@ def responseFieldDict(self):
         return response
 
     def __unicode__(self):
-        return "{0}, {1} IN {2}".format(
+        return "{0}, {1} IN {2} ID {3}".format(
             self.subject.last_name,
             self.subject.first_name,
-            self.external_system.name
+            self.external_system.name,
+            self.record_id
```

Comment:
it looks like 'self.record_id' was added by accident, please remove.

Reply:
ec3aa9077077640eefcc2280e0c19c1d414eaaff